websites/docs-smoke-test/src/content/components/rss-feeds.mdx

```diff
@@ -8,17 +8,6 @@ import { RssFeeds } from '@commercetools-docs/ui-kit'
 
 <Info>The RSS feed is only generated in production builds so this test fails in development mode.</Info>
 
-## Table with one feed
-
-<RssFeeds dataSources={[
-    "https://docs.commercetools.com/merchant-center/releases/feed.xml"
-]}/>
-
-## Table with multiple feeds
-
 <RssFeeds dataSources={[
-    "https://docs.commercetools.com/api/releases/feed.xml",
-    "https://docs.commercetools.com/merchant-center/releases/feed.xml",
-    "https://docs.commercetools.com/custom-applications/releases/feed.xml",
-    "https://docs.commercetools.com/import-api/releases/feed.xml"
+    "/docs-smoke-test/releases/feed.xml"
```

Comment:
```suggestion
    "/docs-smoke-test/releases/feed.xml",
    "/api-docs-smoke-test/releases/feed.xml"
```
having just one feed in a test for multiple feeds is not really testing it.  If it's empty please create a dummy entry in the api docs site to test the aggregation logic.   The feeds are only available in full builds so in dev mode a single feed would not work too.